api/controllers/FlairController.js

```diff
@@ -154,7 +154,7 @@ module.exports = {
 
       for (var i = 0; i < flairs.fcs.length; i++) {
         let fc = flairs.fcs[i];
-        if (!Flairs.validFC(fc) && _.contains(req.user.loggedFriendCodes, fc)) {
+        if (!Flairs.validFC(fc) && flairs.fcs[i] !== req.user.loggedFriendCodes[i]) {
```

Comment:
This is the spam prevention -- friend codes should only be flagged if they are *not* already logged. (If they are already logged, then a modmail has already been sent, so there's no need to send another.)

I didn't do something like `!_.contains(...)` because a modmail should get sent if a user's invalid FC is not the first one. (This would mean a user had an invalid friendcode, changed it to a valid one, and changed it back to the old invalid one for some reason.)